### PR TITLE
Fix broken links and add clarifications in documentation

### PR DESCRIPTION
**Pull request by Evi1Pumpkin**
1. We had a few broken links in our docs. I fixed them.
2. Added some clarifications in documentation related to `remote` and `cloud` backends.

Resolves https://github.com/opentofu/opentofu/issues/1100

## Target Release

1.7.0

### CODE REVIEW

**Review by janosdebugs — APPROVED** [review]
Looks good, but I'm still not sure about the TACOS term. I think I'd prefer the term "cloud".

File: website/docs/language/values/variables.mdx [hunk 1]
```diff
@@ -234,7 +234,7 @@ disclosing the content of one block might imply the content of a sibling block.
 ```
 
 A provider can also
-[declare an attribute as sensitive](/docs/plugin/sdkv2/best-practices/sensitive-state#using-the-sensitive-flag),
+declare an attribute as sensitive,
```
Comment (Yantrio) [inline website/docs/language/values/variables.mdx:237]: maybe we should link to the provider sdk docs here? what do others think?

**Review by Yantrio — CHANGES_REQUESTED** [review]
Looks good to me, just some minor nitpicks.

Also in the future, can you please clarify that you're making other changes on top of fixing links in the PR title and description please? Thanks!

File: website/docs/language/settings/backends/remote.mdx [hunk 2]
```diff
@@ -11,15 +11,15 @@ description: >-
 **We recommend using the [`cloud` built-in integration](/docs/cli/cloud/settings)** instead of this backend. The `cloud` option includes an improved user experience and more features.
 :::
 
-The remote backend is unique among all other OpenTofu backends because it can both store state snapshots and execute operations for TACOS (TF Automation and Collaboration Software) [CLI-driven run workflow](/docs/cloud-docs/run/cli). It used to be called an "enhanced" backend.
+The remote backend is unique among all other OpenTofu backends because it can both store state snapshots and execute CLI-driven run workflow operations for TACOS (TF Automation and Collaboration Software) backends if supported by your TACOS provider. It used to be called an "enhanced" backend.
```
[the remark this answers is not in the document]
Comment (Evi1Pumpkin) [reply website/docs/language/settings/backends/remote.mdx]: I agree. Removed it.

[the remark this answers is not in the document]
Comment (Evi1Pumpkin) [reply website/docs/language/settings/backends/remote.mdx]: Fixed it. Also fixed it through the whole page.

File: website/docs/language/settings/backends/remote.mdx [hunk 3]
```diff
@@ -11,15 +11,15 @@ description: >-
...
-When using full remote operations, operations like `tofu plan` or `tofu apply` can be executed in TACOS (TF Automation and Collaboration Software) run environment, with log output streaming to the local terminal. Remote plans and applies use variable values from the associated remote workspace.
+If your TACOS provider enables full remote operations, you can execute commands such as `tofu plan` or `tofu apply` within the TACOS runtime environment, with log output streaming directly to your local terminal. Remote plans and applies use variable values from the associated remote workspace.
 
-You can also use TACOS (TF Automation and Collaboration Software) with local operations, in which case only state is stored in the remote backend.
+You can also use TACOS with local operations, where only state is stored in the TACOS remote backend.
 
 ## Command Support
 
-The remote backend supports the following OpenTofu commands:
+The remote backend supports the following OpenTofu commands (it's up to the TACOS provider to implement support for each command):
```
[the remark this answers is not in the document]
Comment (Evi1Pumpkin) [reply website/docs/language/settings/backends/remote.mdx]: Agree. Added warning block instead.

File: website/docs/language/settings/tf-cloud.mdx [hunk 4]
```diff
@@ -1,18 +1,18 @@
 ---
 description: >-
-  The nested `cloud` block configures OpenTofu's integration with a cloud backend.
+  The nested `cloud` block configures OpenTofu's integration with a TACOS backend.
```
[the remark this answers is not in the document]
Comment (Evi1Pumpkin) [reply website/docs/language/settings/tf-cloud.mdx]: Agree. After our little discussion, I understood the terminology better and changed it back.

File: website/docs/language/settings/backends/remote.mdx [hunk 5]
```diff
@@ -74,26 +74,6 @@ Because of this, the [`terraform.workspace`](/docs/language/state/workspaces#cur
...
-
-variable "TFC_RUN_ID" {
-  type    = string
-  default = ""
-}
-
-output "remote_execution_determine" {
-  value = "Remote run environment? %{if var.TFC_RUN_ID != ""}Yes%{else}No this is local%{endif}!"
-}
-```
```
[the remark this answers is not in the document]
Comment (Evi1Pumpkin) [reply website/docs/language/settings/backends/remote.mdx:94]: From quick googling, I didn't find any other TACO except TFC that supports this feature. IMO, this is a distinct TFC feature, and we shouldn't have those in the openTofu doc.

[the remark this answers is not in the document]
Comment (DicsyDel) [reply website/docs/language/settings/backends/remote.mdx:94]: @Yantrio In Scalr we have SCALR_RUN_ID

[the remark this answers is not in the document]
Comment (Evi1Pumpkin) [reply website/docs/language/settings/backends/remote.mdx:94]: So what do you think @Yantrio?

[the remark this answers is not in the document]
Comment (Yantrio) [reply website/docs/language/settings/backends/remote.mdx:94]: Okay, let's remove this section for now, I know spacelift doesn't set this variable too. 

We can always re-add it if needed.

File: website/docs/language/values/variables.mdx (hunk 1 above)
Comment (Yantrio) [reply website/docs/language/values/variables.mdx:237]: @opentofu/core-engineers ?

Comment (Evi1Pumpkin) [reply website/docs/language/values/variables.mdx]: So we decided we'll link to the provider SDK docs here

**Review by Yantrio — APPROVED** [review]

**Comment by cam72cam** [comment]
Hi @simaotwx, I've created https://github.com/opentofu/opentofu/issues/1666 to help clear up the confusion.  Many of the top [supporters](https://opentofu.org/supporters/) of OpenTofu build software that makes managing infrastructure described by OpenTofu easier at an organization level.

**Comment by simaotwx** [comment]
@cam72cam Thank you. From what I've understood from reading more of the documentation and the issue it seems like TACOS is not actually a piece of software but rather a category of software that works with OpenTofu as a backend. Initially I thought that TACOS is some self-hosted backend software endorsed or developed by OpenTofu.

**Pull request merged**